### ISSUE TITLE
Dataframe IDs not unique

### ISSUE THREAD

**Issue by j-budler** [geophysics-ubonn/reda#27]
When adding reciprocal and normal data to one data frame the IDs of the data frame are not unique anymore.

**Comment by florian-wagner** [comment]
Can you give a bit more information? You mean when loading normals, assigning IDs and adding reciprocals afterwards?

**Comment by j-budler** [comment]
Example:
container = reda.ERT()
container.import_syscal_txt('data_normal.txt')
container.import_syscal_txt('data_reciprocal.txt', reciprocals=48)
print(container.data.get_value(1, 'R'))

This will give me two values, because the ids of the dataframe are not unique anymore. I know, that I can make them unique myself. But maybe this should be something that is automatically happening.

**Comment by m-weigand** [comment]
I don't think this is feasible. It will be hard to ensure continuity everytime you change a DataFrame by adding/removing rows.

Why exactly do you need this feature? Maybe there is a more robust way, such as .query?

**Comment by j-budler** [comment]
I don't need this feature, because I solved my problem otherwise. I only opened this issue to propose this feature, not to request it.

**Comment by florian-wagner** [comment]
IDs in the data frame now get renumbered on data import (https://github.com/geophysics-ubonn/reda/commit/e5d20fb6601cb09968823041b59ae8199fa71596). So they are always unique, yet not persistent through a processing workflow, which is a different issue and much more complicated.
